idea/testData/inspectionsLocal/mainFunctionReturnUnit/junit4Test.kt

```diff
@@ -0,0 +1,9 @@
+// FIX: Add explicit Unit return type
```

Comment:
Please add `// PROBLEM: <problem text>` to one test with JUnit function and one test with main function